index.html

```diff
@@ -1477,9 +1482,15 @@ <h2>
         </li>
 
         <li>
-Each service endpoint MUST include <code>id</code>,
-<code>type</code>, and <code>serviceEndpoint</code> properties, and
-MAY include additional properties.
+Each service endpoint MAY include an <code>id</code> property.
+The array of service endpoints MUST NOT contain duplicate entries
+with the same <code>id</code>.
+        </li>
+
+        <li>
+Each service endpoint MUST include <code>type</code> and
+<code>serviceEndpoint</code> properties, and MAY include additional
```

Comment:
We need to go through the spec and talk about property-value pairs. `properties` may be confusing to readers... because they're typically the "key" part of a "key-value" pair.

Reply:
It's an RDF term, right? And it's used in many places throughout the spec. So how about we don't address it in this specific PR, but instead raise a new (editorial) issue to see if we can find a better term.

Reply:
Not RDF... that's predicate-object (which will just confuse everyone, so we shouldn't use it). "key-value" is concerning because of the prevalence of "public keys" in the spec. We try to define things using JSON-ish terminology, but we should probably avoid the phrase "key-value" and prefer "property-value" to refer to the data model concept of an associative array... but your suggestion works for me, let's deal with it in another PR.